test/test_distributions.py

```diff
@@ -391,6 +399,24 @@ def get_sp_dist(jax_dist):
     T(dist.LogUniform, 1.0, 2.0),
     T(dist.LogUniform, 1.0, np.array([2.0, 3.0])),
     T(dist.LogUniform, np.array([1.0, 2.0]), np.array([[3.0], [4.0]])),
+    T(
+        dist.MatrixNormal,
+        1.0 * np.arange(6).reshape(3, 2),
```

Comment:
Tests are failing at the `sample` method. Probably the event shape is `(2, 3)`, not (3, 2)?